code/datums/uplink_item.dm

```diff
@@ -760,8 +760,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			Unable to be understood by vampire and changeling agents."
 	reference = "SCS"
 	item = /obj/item/sleeping_carp_scroll
+	refund_path = /obj/item/sleeping_carp_scroll
```

Comment:
Actually not necessary if you do not want your refundable item to be different from the item it grants you. 

Just remove refund_path.

Reply:
Line removed, @variableundefined